### PR TITLE
Fix aircraft not taking off properly

### LINKED ISSUE #17771: If you send planes to rearm with F they don't push already rearmed planes out of airfields

### PR DESCRIPTION
**Pull request by abcdefg30**
Close #17771.

### CODE REVIEW

File: OpenRA.Mods.Common/Activities/Resupply.cs [hunk 1]
```diff
@@ -187,7 +187,7 @@ void OnResupplyEnding(Actor self, bool isHostInvalid = false)
 			{
 				if (wasRepaired || isHostInvalid || (!stayOnResupplier && aircraft.Info.TakeOffOnResupply))
 				{
-					if (self.CurrentActivity.NextActivity == null && rp != null && rp.Path.Count > 0)
+					if (self.CurrentActivity.NextActivity == null && rp != null && rp.Path.Any())
```
Comment (Mailaender) [inline OpenRA.Mods.Common/Activities/Resupply.cs]: Why this change? Avoiding LINQ should be good for performance?

Comment (abcdefg30) [reply OpenRA.Mods.Common/Activities/Resupply.cs]: You're right, I was reflex confusing this with `Count()`.

**Review by Mailaender — APPROVED** [review]

**Review by deleted-user-1 — APPROVED** [review]
Fix works.

**Comment by reaperrr** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/86ffd2229ebf8ac4280e07946a891dbc9ba0dd36...d79fdd0a37cf5602e9acfda0fff060dcbb2dee9d

b698061
